### PR TITLE
Disable extension manager if both black and white listtings are configured

### PR DESCRIPTION
**Pull request by echarles**
Implement https://github.com/jupyterlab/jupyterlab/issues/8165

If both black and white listings are configured, the following is show.

simultaneous_black_white_listings

Documentation has also been updated.

### CODE REVIEW

File: docs/source/user/extensions.rst [hunk 1]
```diff
@@ -259,7 +259,18 @@ the user is free to install any extension. This is the :ref:`default_mode`.
 To bring more security, you or your administrator can enable ``blacklists`` or ``whitelists``
 mode. JupyterLab will check the extensions against the defined listings.
 
-Only one mode at a time is allowed.
+.. warning::
+
+    Only one mode at a time is allowed. If your or your server administrator configures
+    both black and white listings, the extension manager will be disabled
```
Comment (jasongrout) [inline docs/source/user/extensions.rst:265]:
```suggestion
    Only one mode at a time is allowed. If you or your server administrator configures
    both black and white listings, the extension manager will be disabled.
```

File: packages/extensionmanager/src/listings.ts [hunk 2]
```diff
@@ -63,9 +63,12 @@ export class Lister {
           entries: []
         };
         if (data.blacklist_uris.length > 0 && data.whitelist_uris.length > 0) {
-          console.warn(
-            'Simultaneous black and white list are not allowed. Continuing with default mode.'
-          );
+          console.warn('Simultaneous black and white list are not allowed.');
```
Comment (jasongrout) [inline packages/extensionmanager/src/listings.ts:66]:
```suggestion
          console.warn('Simultaneous black and white lists for the extension manager is not allowed.');
```

File: packages/extensionmanager/src/widget.tsx [hunk 3]
```diff
@@ -587,6 +587,27 @@ export class ExtensionView extends VDomRenderer<ListModel> {
...
+          <div>
+            The extension manager is disabled. Please contact your system
+            administrator to verify the listings configuration.
+          </div>
+          <div>
+            <a
+              href="https://jupyterlab.readthedocs.io/en/stable/user/extensions.html"
+              target="_blank"
+            >
+              Read more on the JupyterLab documentation.
```
Comment (jasongrout) [inline packages/extensionmanager/src/widget.tsx:605]:
```suggestion
              Read more in the JupyterLab documentation.
```

**Comment by echarles** [comment]
> Thanks! I suggested a few changes.

Thx a lot for thoses fixes. I have merged them.

> Also, can you run pngquant on the png files in the listings directory? See the guidelines at https://jupyterlab.readthedocs.io/en/stable/developer/contributing.html#screenshots-and-animations

Done.

**Pull request merged**